src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.0.0-rc9.14
+
+Released 2023-Feb-24
```

Comment:
we never used to do this, but ideally we should Bumps Otel dependency to 1.4.0 as well.?

Reply:
+1 There's definitely been confusion over what version non-core components use. I don't think this solves any potential confusion completely, but I this this is a small step in the right direction.

Reply:
Ok just brainstorming here... I don't know if I like this idea yet.

Could we start a new versioning scheme for non-core components? Like:

`1.0.0-rc140`
`1.0.0-rc150.alpha.1`
`1.0.0-rc150.alpha.2`
`1.0.0-rc150.beta.1`
`1.0.0-rc150.rc.1`
`1.0.0-rc150`

I'm pretty sure this would get the correct semver ordering from nuget.

Reply:
For this release, lets stick with the current (but add an explicit changelog entry about updating otel to 1.4).
For the next release, we can discuss more better ways to handle this. (its possible the problem may got away if spec marks stable for http conventions by April..)

Reply:
https://semvercompare.azurewebsites.net/?version=1.0.0-rc140&version=1.0.0-rc150.alpha.1&version=1.0.0-rc150.alpha.2&version=1.0.0-rc150.beta.1&version=1.0.0-rc150.rc.1&version=1.0.0-rc150&version=1.0.0-rc999

Reply:
> For the next release, we can discuss more better ways to handle this.

Good by me to wait until 1.5.0. Though we will likely still need to consider making better for GrpcNetClient/SqlClient instrumentation. Unless we plan to move these to the contrib.